pkg/cli/cve_cmd.go

```diff
@@ -131,8 +135,51 @@ type cveFlagVariables struct {
 	fixedFlag       *bool
 }
 
+func checkExtEndPoint(serverURL string) bool {
+	client := resty.New()
+
+	extEndPoint, err := combineServerAndEndpointURL(serverURL, fmt.Sprintf("%s%s",
+		constants.RoutePrefix, constants.ExtOciDiscoverPrefix))
+	if err != nil {
+		return false
+	}
+
+	// nolint: gosec
+	resp, err := client.R().Head(extEndPoint)
+	if err != nil {
+		return false
+	}
+
+	if resp.StatusCode() == http.StatusOK {
+		searchEndPoint, err := combineServerAndEndpointURL(serverURL, constants.ExtSearchPrefix)
```

Comment:
`extentions` and `search` endpoints are needed for both the old logic and the new logic making a single graphql call.
We should check the new query to differentiate them by checking `ImageList` is available.